### DISCUSSION TITLE
Unsubscribe option from your mailing list

### DISCUSSION THREAD

**Discussion by satyaakam** [dailydotdev/daily#388] [General]
Hi,

Unsubscribe option on the mailer lands back to the home page of your website instead of unsubscribe option.

thanks
-Satya

**Comment by nimrodkra** [comment]
Hi @satyaakam,

Thanks for reaching out here! Sorry for the inconvenience. We looked into this issue and indeed found that one of our recent emails was misconfigured. If you would like to unsubscribe, can you please email us at support@daily.dev? We'll take it from there and unsubscribe you.
